common/djangoapps/track/views/__init__.py

```diff
@@ -71,7 +71,7 @@ def user_track(request):
 
     return HttpResponse('success')
 
-
+@trackit
```

Comment:
Let's leave this to the analytics team to add, once we're done writing the supporting code.